packages/kotti-ui/source/kotti-field/tests.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable vue/one-component-per-file */
```

Comment:
This is turned off globally in this PR, so this line should be redundant

Reply:
not globally, for test files. But yes, good point